build_projects/dotnet-host-build/CompileTargets.cs

```diff
@@ -171,7 +171,9 @@ public static BuildTargetResult CompileCoreHost(BuildTargetContext c)
         {
             var hostVersion = c.BuildContext.Get<HostVersion>("HostVersion");
             var configuration = c.BuildContext.Get<string>("Configuration");
-            string rid = c.BuildContext.Get<string>("TargetRID");
+            string rid = DotNetCli.Stage0.GetRuntimeId();
```

Comment:
e.g.

```
if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
{
...
    string ridMacro = $"-DCLI_CMAKE_RUNTIME_ID:STRING=win10-{platform.ToLower()}";
...
}
```

instead of
```
if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
{
...
    rid = c.BuildContext.Get<string>("TargetRID");
    string ridMacro = $"-DCLI_CMAKE_RUNTIME_ID:STRING={rid}";
...
}
```